02-visualization.Rmd

```diff
@@ -242,14 +242,14 @@ The simplest of the 5NG are *scatterplots*, \index{scatterplots} also called *bi
 1. `dep_delay`: departure delay on the horizontal "x" axis and
 1. `arr_delay`: arrival delay on the vertical "y" axis
 
-for Alaska Airlines flights leaving NYC in 2013. This requires paring down the data from all 336,776 flights that left NYC in 2013, to only the 714 *Alaska Airlines* flights that left NYC in 2013. We do this so our scatterplot will involve a manageable 714 points, and not an overwhelmingly large number like 336,776. To achieve this, we'll take the `flights` data frame, filter the rows so that only the 714 rows corresponding to Alaska Airlines flights are kept, and save this in a new data frame called `alaska_flights` using the `<-` *assignment* operator: \index{operators!assignment (<-)} 
+for Alaska Airlines flights leaving NYC in 2013. This requires paring down the data from all `r comma(nrow(nycflights13::flights))` flights that left NYC in 2013, to only the `r nrow(nycflights13::flights %>% filter(carrier == "AS"))` *Alaska Airlines* flights that left NYC in 2013. We do this so our scatterplot will involve a manageable `r nrow(nycflights13::flights %>% filter(carrier == "AS"))` points, and not an overwhelmingly large number like `r comma(nrow(nycflights13::flights))`. To achieve this, we'll take the `flights` data frame, filter the rows so that only the `r nrow(nycflights13::flights %>% filter(carrier == "AS"))` rows corresponding to Alaska Airlines flights are kept, and save this in a new data frame called `alaska_flights` using the `<-` *assignment* operator: \index{operators!assignment (<-)} 
```

Comment:
Let's discuss the above in our meeting. In particular about adding computations to the build time of the book.

Reply:
See my remark about inline outputting the same value x3 or more in `04-tidy.Rmd`. Apply this logic for rest of PR

Reply:
For cases like this where code is run after inline r computation. Create a duplicate `echo=FALSE, purl=FALSE` block just above text, with comment `# This redundant code is used for text output purposes`